src/routes/single-page-app.test.js

```diff
@@ -1,43 +0,0 @@
-import { init } from "../app";
```

Comment:
Was this change on purpose?

Reply:
Yes, git just didn't understand that it's a file move. I was moving this file to src/app.test.js since the test imports app and then runs some tests.